build_scripts/shared_versions.sh

```diff
@@ -134,6 +137,14 @@ version_is_supported() {
     fi
 }
 
+# Ensure PG version matching is performed only based upon MAJOR version.
+#
+# This is a bit simplistic, but the system will otherwise break when
+# building a version of PG which specifies a minor or patch version.
+major_version_only() {
+    "$(awk -F'.' '{print $1}' <<<"$1")"
```

Comment:
nit / not important, but you could drop a fork with native bash functionality:
```suggestion
    echo "${1%%.*}"
```

Reply:
Thanks Brent! Done.